### ISSUE TITLE
Legend stack toggle buttons are missing "aria-label" attributes

### ISSUE THREAD

**Issue by AleksueiR** [fgpv-vpgf/fgpv-vpgf#2517]
All buttons must have discernible text - for icon buttons it means having `aria-label` attributes.

![](https://i.imgur.com/G18bleK.png)
